discover/deployment/articles/02-configuring-liferay/02-document-repository.markdown

```diff
@@ -222,10 +222,11 @@ You must include this property in `system-ext.properties`:
     org.xml.sax.driver=com.sun.org.apache.xerces.internal.parsers.SAXParser
 
 Other app servers also need this configuration if they don't contain a
-`SAXParser`. 
+`SAXParser`. Remember to place the `system-ext.properties` in your @product@
+installation's `/WEB-INF/classes` directory.
 
```

Comment:
I haven't yet verified in other App servers (I can test WAS soon), but system-ext isn't picked up when in Liferay Home. Try it out; normally you'll see a list of properties that Liferay is picking up, and if system-ext is not in WEB-INF/classes, then it won't be picked up. Not sure why since portal-ext is picked up, but that's what I found.